src/client/components/organisms/UserCard/UserCard.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FaEye as ViewIcon,
+  FaPen as EditIcon,
+  FaTrashAlt as DeleteIcon,
+} from 'react-icons/fa';
+
+import ListReadableFields from '../../molecules/ListReadableFields/ListReadableFields';
+import ReadableField from '../../atoms/ReadableField/ReadableField';
+import Button from '../../atoms/Button/Button';
+
+import './UserCard.scss';
+import { prototype } from 'enzyme-adapter-react-16';
```

Comment:
This import is already in use ?

Reply:
This is still in progress.
And that was a rare mistake :rofl: